### ISSUE TITLE
copy to clipboard in graph menu does not work (Trac #559)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#691]
right clicking on a graph brings up a pop up menu with a number of options.  The top 3 relate to printing, saving, and copying to clipboard.  Choosing the last generates no indication of an error but does not in fact copy anything to the system clipboard.

Migrated from http://trac.sasview.org/ticket/559
```json
{
    "status": "closed",
    "changetime": "2016-06-19T15:31:48",
    "_ts": "2016-06-19 15:31:48.311400+00:00",
    "description": "right clicking on a graph brings up a pop up menu with a number of options.  The top 3 relate to printing, saving, and copying to clipboard.  Choosing the last generates no indication of an error but does not in fact copy anything to the system clipboard.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2016-04-15T14:15:39",
    "component": "SasView",
    "summary": "copy to clipboard in graph menu does not work",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.0.0",
    "owner": "butler",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2016/04/16 18:23:26`: 
* **butler** changed _comment0 from:

> This error was apparently introduced in July 2015 as we were trying to fix the wx 3.0 and new matplotlib issues with saving, printing etc of plots. The copy to clipboard call was implemented within a try except but forgot to pass the canvas to plot argument required by the copy to clipboard function hence failing silently (error statement sent to stdout I guess (was not in !SasView console output but was in eclipse console.
> 
> should be tested once we have distribution builds before closing

to:

> 1466350213673882


* **butler** commented:

This error was apparently introduced in July 2015 as we were trying to fix the wx 3.0 and new matplotlib issues with saving, printing etc of plots. The copy to clipboard call was implemented within a try except but forgot to pass the canvas to plot argument required by the copy to clipboard function hence failing silently with the error statement sent to stdout I guess (was not in !SasView console output but was in eclipse console).

should be tested once we have distribution builds before closing


* **butler** changed owner from "" to "butler"
* **butler** changed status from "new" to "accepted"

**Comment by butlerpd** [comment]
Trac update at `2016/06/19 15:31:48`: 
* **butler** commented:

Is working in versin 4.0alpha2 for windows downloaded from !GitHub.  Is such an obvious and simple fix that I am now going to close the ticket.


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "accepted" to "closed"
